open-api/rest-catalog-open-api.yaml

```diff
@@ -212,6 +212,34 @@ paths:
           schema:
             type: string
           example: "accounting%1Ftax"
+        - name: pageToken
+          in: query
+          description:
+            Allows clients to make use of pagination by signaling to the service that they would
+            prefer requests to be paginated to a “reasonable” number of results.
+            
+            New Clients always start the request by sending a required empty “pageToken” e.g. GET /tables?continuationToken=””
```

Comment:
1. if the query parameter value is empty, I think it does not contain double quotes.
2. seems like we want to distinguish the behavior of returning everything vs returning paginated by passing the initial empty page token. That makes sense from backwards compatibility perspective, but I think the client should not need to always start the request with the empty query parameter. If it is not specified, the service should have the freedom to return all results or a paginated result, and this still keeps backwards compatibility.

Reply:
I think you are right, it seems for query parameter value that is emtpy I believe we dont need to put any value.
https://stackoverflow.com/questions/4557387/is-a-url-query-parameter-valid-if-it-has-no-value